libraries/AP_RangeFinder/AP_RangeFinder_VL53L1X.cpp

```diff
@@ -86,14 +86,32 @@ bool AP_RangeFinder_VL53L1X::check_id(void)
     return true;
 }
 
+bool AP_RangeFinder_VL53L1X::reset(void) {
+    if (!write_register(SOFT_RESET, 0x00)) {
+        return false;
+    }
+    hal.scheduler->delay_microseconds(100);
+    if (!write_register(SOFT_RESET, 0x01)) {
+        return false;
+    }
+    hal.scheduler->delay(1000);
```

Comment:
We discussed this on the dev call. We thought it would be better to use a state variable, and not wait a full second here. Instead just mark it unhealthy until the init can complete